package.json

```diff
@@ -65,8 +65,7 @@
   },
   "lint-staged": {
     "*.js": [
-      "eslint --format=codeframe --fix",
-      "git add"
+      "eslint --format=codeframe --fix"
```

Comment:
Might also be worth introducing something for prettier? https://github.com/prettier/prettier#pre-commit-hook